src/config.ts

```diff
@@ -0,0 +1,56 @@
+import {Observable, Subject} from 'rxjs/Rx';
+
+export class AllOfUsConfig {
+
+  private config = new Subject<AllOfUsConfig>();
+  public config$:Observable<AllOfUsConfig> = this.config.asObservable();
+
+  private workspaceNamespace:string;
+  private workspaceId:string;
+  private apiHost:string;
+  private cdrVersionBigqueryDataset:string;
+  private cdrVersionCloudProject:string;
+  private bucketName:string;
+
+  public setJson(jsonString:string):void
+  {
```

Comment:
I have a visceral reaction to this bracket placement. tslint hopefully would make this a non-issue, but I'll at least say that most of our workbench UI code puts this on the previous line